openpype/hosts/blender/scripts/build_workfile.py

```diff
@@ -398,18 +400,18 @@ def build_layout(project_name, asset_name):
 
         # Link loaded containers to layout collection
         for c in containers:
-            layout_instance.datablock_refs[0].datablock.children.link(
-                c.outliner_entity
-            )
-            bpy.context.scene.collection.children.unlink(c.outliner_entity)
+            if not isinstance(c.outliner_entity, bpy.types.Collection):
+                continue
+            if c.outliner_entity not in layout_container.children.values():
+                layout_container.children.link(c.outliner_entity)
+            if c.outliner_entity in bpy.context.scene.collection.children.values():
```

Comment:
```suggestion
            if (
                    c.outliner_entity
                    in bpy.context.scene.collection.children.values()
            ):
```
This line is too long.